pyinterprod/uniprot/unirule.py

```diff
@@ -707,3 +707,221 @@ def update_signatures(filepath: str, uri: str):
     con.commit()
     cur.close()
     con.close()
+
+
+MAX_DOM_BY_GROUP = 20
+DOM_OVERLAP_THRESHOLD = 0.3
+REPR_DOM_DATABASES = ["H", "J", "M", "R", "N"]
+DC_STATUSES = {
+    "S": "CONTINUOUS",
+    "N": "N_TERMINAL_DISC",
+    "C": "C_TERMINAL_DISC",
+    "NC": "NC_TERMINAL_DISC"
+}
+
+
+def get_repr_domains(ora_url: str, output: str = "repr_domains.tsv"):
+    con = oracledb.connect(ora_url)
+    cur = con.cursor()
+    cur.execute(
+        """
+        SELECT PROTEIN_AC, H.METHOD_AC, MODEL_AC, H.DBCODE, 
```

Comment:
Unnecessary (not used):

```suggestion
        SELECT PROTEIN_AC, H.METHOD_AC, MODEL_AC,
```